src/ExtensionGallery/Startup.cs

```diff
@@ -1,109 +1,63 @@
 ﻿using System;
-using Microsoft.AspNet.Http;
 using System.Linq;
-using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Diagnostics;
-using Microsoft.AspNet.Hosting;
-using Microsoft.AspNet.Routing;
-using Microsoft.AspNet.StaticFiles;
-using Microsoft.Framework.Configuration;
-using Microsoft.Framework.DependencyInjection;
-using Microsoft.Framework.Logging;
-using Microsoft.Framework.Runtime;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.PlatformAbstractions;
+using Newtonsoft.Json.Serialization;
 
 namespace ExtensionGallery
 {
 	public class Startup
 	{
-		public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
+		public Startup(IHostingEnvironment env)
 		{
             // Setup configuration sources.
-            Configuration = new ConfigurationBuilder(appEnv.ApplicationBasePath)
-				.AddJsonFile("config.json")
-				.AddEnvironmentVariables().Build();
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+                .AddEnvironmentVariables();
+            Configuration = builder.Build();
 		}
 
 		public IConfiguration Configuration { get; set; }
 
 		// This method gets called by the runtime.
 		public void ConfigureServices(IServiceCollection services)
 		{
-			// Add EF services to the services container.
-			//services.AddEntityFramework(Configuration)
-			//    .AddSqlServer()
-			//    .AddDbContext<ApplicationDbContext>();
-
-			//// Add Identity services to the services container.
-			//services.AddDefaultIdentity<ApplicationDbContext, ApplicationUser, IdentityRole>(Configuration);
-
 			// Add MVC services to the services container.
-			services.AddMvc();
+			services.AddMvc()
+                .AddJsonOptions(options => options.SerializerSettings.ContractResolver = new DefaultContractResolver());
```

Comment:
MVC now serializes JSON with camel case names by default, per https://github.com/aspnet/Announcements/issues/194

Leaving camelcase as default causes a bunch of stuff to break in the angular code.  The better fix is probably to rewrite the JS to assume camelcase, but that's a hassle, so I just used this workaround for now.